frontend/containers/LazyLoader/LazyLoader.jsx

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+import {h, Component} from 'preact'
+import {connect} from 'preact-redux'
+import proptypes from 'proptypes'
+import {bindActionCreators} from 'redux'
+import {connectHelper} from 'lib/util'
+
+import * as appActions from 'actions/appActions'
+
+class LazyLoader extends Component {
+  static propTypes = {
```

Comment:
Could you follow the same spacing convention as used in the proptypes definition of other components? (Also period at the end of the sentences, if possible)